config.json

```diff
@@ -572,7 +572,13 @@
       "core": false,
       "difficulty": 7,
       "slug": "word-search",
-      "topics": null,
+      "topics": [
+        "string",
+        "words",
```

Comment:
This isn't a topics in the topics list and I'm not sure it makes sense as a topic? `words` doesn't seem like a language concept to me. But feel free to disagree :)